### DISCUSSION TITLE
IECWind results doesn't reflect the input parameters in the IEC.ipt input file

### DISCUSSION THREAD

**Discussion by LaurenceWETI** [OpenFAST/openfast#1580] [Q&A]
Dear @jjonkman, 

As,I could not run  [IECWind_v5.01.02](https://www.nrel.gov/wind/nwtc/assets/downloads/IECWind/IECWind_v5.01.02.exe) on my pc, I downloaded   [your recompiled version](https://drive.google.com/file/d/17IJ0vPsULTeYGOd9HhS6n5SNbWH8CH1O/view?usp=sharing) and I recompiled the source code, too. The recompiled IECWind.exe versions are running now. However, the output files didn't reflect the changes that I've done in the input file IEC_ipt. 
Could you send me please a previous version of IECWind, e.g. IECWind_v4?

Best regrds

**Comment by jjonkman** [comment]
Dear @LaurenceWETI,

I could send you an older version of IECWind, but generally don't promote downgrading to an older version.  I'm not aware of an issue with the recompiled version of IECWind that I shared, but if you are recompiling yourself anyway, I would suggest debugging why the code is not working as expected.  Perhaps you didn't save your input file after changing it?

Best regards,
